xreds/dataset_provider.py

```diff
@@ -58,27 +58,30 @@ def get_dataset(self, dataset_id: str) -> xr.Dataset:
 
             if dataset_path.endswith('.nc'):
                 ds = xr.open_dataset(dataset_path)
-            elif '.zarr' in dataset_path:
-                # TODO: Enable S3  support
-                # mapper = fsspec.get_mapper(dataset_location)
-                ds = xr.open_zarr(dataset_path, consolidated=True)
-            elif dataset_path.endswith('.json'):
+            elif dataset_path.endswith('.json') or dataset_path.endswith('.nc.zarr'):
```

Comment:
This is a hack based on the nos kerchunked files, we should prob maybe just have this in the config